ifrs17-template/Report/ReportEyCases.ipynb

```diff
@@ -47,7 +47,17 @@
         {
             "cell_type": "code",
             "source": [
-                "#!import \"../Initialization/InitSystemorphToMemory\""
+                "#!import \"../Initialization/InitSystemorphUtils\""
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "await InitializeDataNodesAsync(DataSource, pathToImport, new string []{\"FR\"});",
+                "\nawait InitializeTransactionalDataAsync(DataSource, pathToImport, new string[] {\"FR\"});"
```

Comment:
API is better with an extension method:
DataSource.InitializeReportingNodeAsync("../../Files/","FR")
also, you can provide the ReportingNode as params rather than array => check GetMessage in Validations Notebook

Reply:
Yes, you are right. Though it should extend the import variable rather than the Data source, otherwise passing the import by an alias would be truly awkward. I rewrote them all now as static extensions of IImportVariable